src/cookbook/ScottPlot.Cookbook/Recipes/Plottable/Crosshair.cs

```diff
@@ -116,6 +116,49 @@ public void ExecuteRecipe(Plot plt)
         }
     }
 
+    public class CrosshairCustomLabelFormatter : IRecipe
+    {
+        public string Category => "Plottable: Crosshair";
+        public string ID => "crosshair_customlabelformatting";
+        public string Title => "Custom Axis Label";
+        public string Description =>
+            "For ultimate control over crosshair label format you can create " +
+            "a custom formatter function and use that to convert positions to labels. " +
+            "This allows logic to be used to format crosshair labels.";
+
+        public void ExecuteRecipe(Plot plt)
+        {
+            plt.Title("Crosshair with Custom Label Formmater");
+            plt.XLabel("Horizontal Axis");
+            plt.YLabel("Vertical Axis");
+
+            plt.AddSignal(ScottPlot.DataGen.Sin(51));
+            plt.AddSignal(ScottPlot.DataGen.Cos(51));
+
+            // create a custom formatter as a static class
+            static string customTickFormatter(double position)
+            {
+                if (position == 0)
+                    return "zero";
+                else if (position > 0)
+                    return $"+{position:F2}";
+                else
+                    return $"({Math.Abs(position):F2})";
+            }
+
+            // use the custom formatter for horizontal and vertical tick labels
+            plt.XAxis.TickLabelFormat(customTickFormatter);
+            plt.YAxis.TickLabelFormat(customTickFormatter);
+
+            // add a crosshair
+            var ch = plt.AddCrosshair(20, 0);
+
+            // use the custom formatter for crosshair labels
+            ch.ManualStringFormatterX = customTickFormatter;
+            ch.ManualStringFormatterY = customTickFormatter;
```

Comment:
Again on the consistency argument, it's a little weird that for one of them you pass in a formatter delegate to a function and for the other you set a property.

I think that setting a property is probably the better API, but if we want consistency that would mean changing the axis code.

Reply:
The consistency argument is also present in the original API. I think changing the axis code is a good way.